analyzer/emeraldtokens/emerald_tokens.go

```diff
@@ -0,0 +1,194 @@
+package emeraldtokens
```

Comment:
We're planning to make runtime analyzers and tables more agnostic wrt the specific runtime, and focus them around sdk modules instead. Can we make this into `evm_tokens`? And in general do a search for `[Ee]merald` and see what can be meaningfully and easily renamed or generalized. Otherwise, we'll have to come back and do it in a couple weeks for Sapphire.

Renaming the emerald_tokens DB table (and adding a `runtime` column) is probably out of scope for this PR.

Reply:
changing, also changing in #296

Reply:
renamed